core/domain/cve.go

```diff
@@ -0,0 +1,9 @@
+package domain
+
+type CVE struct {
+	ImageID            string
+	SBOMCreatorVersion string
+	CVEScannerVersion  string
+	CVEDBVersion       string
+	Content            []byte
```

Comment:
question: what does this CVE object contain?

Is it a list of individual vulnerabilities?

Reply:
this is the CVE manifest in JSON format straight from Grype